vaadinfrontend/src/main/java/life/qbic/datamanager/views/support/experiment/ExperimentListComponent.java

```diff
@@ -1,13 +1,11 @@
-package life.qbic.datamanager.views.projects.project.experiments;
+package life.qbic.datamanager.views.support.experiment;
 
```

Comment:
This class will be replaced by the itemCollection down the line? If so you can remove it 👍